test/src/navigation_middleware_test.dart

```diff
@@ -220,8 +220,8 @@ void main() {
       NavigationMiddleware(currentState: navigatorState)
           .call(store, NavigateToAction.push('/name'), (_) {});
 
-      expect(NavigatorHolder.state.currentPath, '/name');
```

Comment:
Please keep those test, so that we are sure we will not introduce breaking changes